plugins/GeoIp2/GeoIP2AutoUpdater.php

```diff
@@ -255,10 +260,10 @@ public static function unzipDownloadedFile($path, $dbType, $unlink = false)
             // delete the existing GeoIP database (if any) and rename the downloaded file
             $oldDbFile = LocationProviderGeoIp2::getPathForGeoIpDatabase($dbFilename);
             if (file_exists($oldDbFile)) {
-                unlink($oldDbFile);
+                @unlink($oldDbFile);
```

Comment:
we already ignored below `@rename //In case the $tempfile cannot be renamed, we copy the file.` but not here the unlink which also may fail but then the `copy` succeeds.